src/AppBundle/Procuration/ProcurationProxyMessageFactory.php

```diff
@@ -40,4 +41,17 @@ public function createProxyFoundMessage(Adherent $procurationManager, Procuratio
 
         return $message;
     }
+
+    public function createProxyReminderMessage(ProcurationRequest $request): ProcurationProxyReminderMessage
+    {
+        $url = $this->urlGenerator->generate('app_procuration_my_request', [
+            'id' => $request->getId(),
+            'token' => $request->generatePrivateToken(),
+        ]);
+
+        $message = ProcurationProxyReminderMessage::create($request, $request->getFoundProxy(), $url, $request->getFoundBy());
```

Comment:
I know, it's just that `ProcurationProxyFoundMessage` and `ProcurrationProxyCancelledMessage` are done like this, so I did it also.